### PR TITLE
add jwt sig validation to changelog

### PR DESCRIPTION
**Pull request by letmaik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - JWT key auto-refresh, can be enabled by providing `"auto_refresh": true` and `"ca_cert_name": "..."` in `set_jwt_issuer` proposal.
   - Auto-refresh is currently only supported for providers following the OpenID Connect standard where keys are published under the `/.well-known/openid-configuration` path of the issuer URL.
   - `ca_cert_name` refers to a certificate stored with a `set_ca_cert` proposal and is used to validate the TLS connection to the provider endpoint.
+- JWT signature validation, can be enabled with the ``require_jwt_authentication`` endpoint property.
```
Comment (jumaffre) [inline CHANGELOG.md:15]: I'd suggest including the PR # for entries in the changelog for this release.

**Review by jumaffre — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (letmaik) [reply CHANGELOG.md]: Done.

**Comment by achamayou** [comment]
@letmaik can you run `./scripts/ci-checks.sh -f`.

**Comment by letmaik** [comment]
We should have a bot that does it for us...

**Pull request merged**